src/main/java/frc/robot/subsystems/conveyor/Conveyor.java

```diff
@@ -0,0 +1,128 @@
+package frc.robot.subsystems.conveyor;
+
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+import com.revrobotics.ColorMatch;
+import com.revrobotics.ColorMatchResult;
+import edu.wpi.first.wpilibj.DriverStation;
+import edu.wpi.first.wpilibj.I2C;
+import edu.wpi.first.wpilibj.SPI;
+import edu.wpi.first.wpilibj.util.Color;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Ports;
+
+import java.util.Deque;
+import java.util.LinkedList;
+import com.revrobotics.ColorSensorV3;
+
+public class Conveyor extends SubsystemBase {
+    private static Conveyor INSTANCE = null;
+    private final WPI_TalonSRX motor = new WPI_TalonSRX(Ports.Conveyor.AUX);
+    private DriverStation.Alliance color;
+    private int cargoCount;
+    private final Deque<String> position = new LinkedList<>();
+    private final ColorSensorV3 colorSensorIntake = new ColorSensorV3(I2C.Port.kMXP);
+    private final ColorSensorV3 colorSensorShooter = new ColorSensorV3(I2C.Port.kOnboard);
+    ColorMatch match = new ColorMatch();

```

Comment:
Rename to `colorMatch` or `colorMatcher`, "match" has multiple definitions, especially in FRC.

Reply:
also, private final